docs/main/heat-pumps.md

```diff
@@ -44,41 +44,42 @@ More information about the sources behind specifications of heat pump ground can
+* **Hydrogen hybrid heat pumps**: [Documentation hybrid heat pump (hydrogen) - space heating](https://github.com/quintel/etdataset-public/blob/master/nodes_source_analyses/energy/households/households_space_heater_hybrid_hydrogen_heatpump_air_water_electricity.converter.xlsx), and [Documentation hybrid heat pump (hydrogen) - hot water](https://github.com/quintel/etdataset-public/blob/master/nodes_source_analyses/energy/households/households_water_heater_hybrid_hydrogen_heatpump_air_water_electricity.converter.xlsx)
+* **Oil hybrid heat pumps**: [Documentation hybrid heat pump (oil) - space heating](https://github.com/quintel/etdataset-public/blob/master/nodes_source_analyses/energy/households/households_space_heater_hybrid_crude_oil_heatpump_air_water_electricity.xlsx), and [Documentation hybrid heat pump (oil) - hot water](https://github.com/quintel/etdataset-public/blob/master/nodes_source_analyses/energy/households/households_water_heater_hybrid_crude_oil_heatpump_air_water_electricity.xlsx)
 
 ### Threshold-COP
 
-The efficiency of hybrid heat pumps (HHPs) is dependent on the ambient temperature and is depicted by the coefficient of performance (COP). The COP becomes lower as the outside temperature decreases. In the ETM it is possible to set the COP for which the HHP must switch between electricity and gas. You can choose a setting that is most financially attractive for the consumer, but you can also choose a setting that produces less impact on the electricity network
+The efficiency of hybrid heat pumps (HHPs) depends on the ambient temperature and is represented by the coefficient of performance (COP). The COP becomes lower as the outside temperature decreases. In the ETM it is possible to set the so-called _threshold COP_, i.e. the COP below which the HHP switches from electricity to its other fuel source (network gas, hydrogen or oil). You can choose a setting that is most financially attractive for the consumer, but you can also choose a setting that reduces the impact on the electricity network. Currently, you can set the threshold COP for space heating for either gas-based HHPs (both network gas and hydrogen) or for the oil HHP. Additionally, the threshold COP for water heating can be adjusted as well for completeness sake, but note that HHPs typically use the fuel part to provide hot water. For this reason, the default water heating COP is set at the maximum value of 6.0, so that the heat pump part is never enabled.
 
-![](/img/docs/20200401_threshold_COP_sliders.png)
+![](/img/docs/20240405_hhp_threshold_cop_sliders.png)
```

Comment:
@KoenvanB or @kaskranenburgQ could you use a download figure instead of a screenshot?